### PR TITLE
Update `pathology_nuclei_segmentation_classification` benchmark

### PR DESCRIPTION
**Pull request by KumoLiu**
Part of https://github.com/Project-MONAI/model-zoo/issues/338 .

### Description

- [x] Retrain pathology_nuclei_segmentation_classification on A100.

- [x] Update figures with URL.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [x] In-line docstrings updated.
- [x] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Comment by KumoLiu** [comment]
Update: waiting for this [PR](https://github.com/Project-MONAI/MONAI/pull/6325) in core.

File: models/pathology_nuclei_segmentation_classification/docs/README.md [hunk 1]
```diff
@@ -94,21 +94,21 @@ python -m monai.bundle run --config_file configs/train.json --network_def#pretra
...
 ```
 
 - Run second stage
 
 ```
-torchrun --standalone --nnodes=1 --nproc_per_node=2 -m monai.bundle run --config_file "['configs/train.json','configs/multi_gpu_train.json']" --train#dataloader#batch_size 4 --network_def#freeze_encoder false --network_def#pretrained_url None --stage 1
+torchrun --standalone --nnodes=1 --nproc_per_node=2 -m monai.bundle run --config_file "['configs/train.json','configs/multi_gpu_train.json']" --train#dataloader#batch_size 4 --network_def#freeze_encoder False --network_def#pretrained_url None --stage 1
 ```
 
 Override the `train` config to execute evaluation with the trained model:
```
Comment (yiheng-wang-nv) [inline models/pathology_nuclei_segmentation_classification/docs/README.md]: May need to add some descriptions on:
1. what is the result of this evaluation, and what is the difference between it and the `Binary Dice` mentioned above.
2. how to calculate the `Binary Dice`, `PQ` and `F1d`.

**Review by Nic-Ma — APPROVED** [review]
Please add the non-deterministic description in README, others look good to me.

Thanks.

File: models/pathology_nuclei_segmentation_classification/configs/metadata.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_hovernet_20221124.json",
-    "version": "0.1.4",
+    "version": "0.1.5",
     "changelog": {
+        "0.1.5": "update benchmark on A100",
         "0.1.4": "adapt to BundleWorkflow interface",
         "0.1.3": "add name tag",
         "0.1.2": "update the workflow figure",
```
Comment (yiheng-wang-nv) [inline models/pathology_nuclei_segmentation_classification/configs/metadata.json:8]: could you change the monai version into rc4

File: models/pathology_nuclei_segmentation_classification/docs/README.md (hunk 1 above)
Comment (KumoLiu) [reply models/pathology_nuclei_segmentation_classification/docs/README.md]: Thanks, have updated!

**Review by yiheng-wang-nv — APPROVED** [review]

**Pull request merged**